src/pages/Authentication.js

```diff
@@ -0,0 +1,48 @@
+import React, { memo, useState } from 'react';
+import { Redirect, useRouteMatch, Link } from 'react-router-dom';
+
+import { useDispatch, useSession, useStatus } from 'hooks';
+import AuthenticationForm from 'components/user/AuthenticationForm';
+import { login, register } from 'state/actions/userActions';
+import routes from 'constants/routesPaths';
+
+function AuthenticationPage() {
+  const { authenticated } = useSession();
+
+  const isLogin = !!useRouteMatch(routes.login);
+  const action = isLogin ? login : register;
+  const onSubmit = useDispatch(action);
+  const [values, setValues] = useState({ email: '', password: '' });
```

Comment:
I think it will be better if you split those values on 2 variables. I think any form will use them as separated values